SDK/AppCenter/Microsoft.AppCenter.Shared/WrapperSdkInfo.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.AppCenter
+{
+    public class WrapperSdkInfo
```

Comment:
In both Java and Apple SDK it's only WrapperSdk class name.
Also both SDKs consider it a model (models sub package in java, model folder in Apple).
Can we align better?